### ISSUE TITLE
#1474369117------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1843]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<span class="title"><slot></slot></span>`

<strong>Element Location: </strong>calcite-chip[value="TRACK_ID"],.title

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\1474369117_All page content should be contained by landmarks.png`
